src/bio/bio_xstream.c

```diff
@@ -98,7 +98,7 @@ bio_spdk_env_init(void)
 	 * logging by setting level to ERROR.
 	 */
 	opts.env_context = "--log-level=lib.eal:4 --log-level=lib.user1:4"
-		" --no-telemetry";
+		" --log-level=lib.pmd:4 --no-telemetry";
```

Comment:
I'm pretty sure this should just be `pmd:4`... https://github.com/DPDK/dpdk/blob/main/lib/eal/common/eal_common_log.c#L356

Reply:
thanks